src/assets/css/style.css

```
@@ -188,7 +189,7 @@ p {
   -moz-background-clip: padding;
   -webkit-background-clip: padding-box;
   background-clip: padding-box;
-  background: #2d7788;
+  background: linear-gradient(135deg, #667eea, #764ba2);
```

Biome (2.1.2)

[error] Unexpected shorthand property background after background-clip



(lint/suspicious/noShorthandPropertyOverrides)